scripts/auto_mesh.py

```diff
@@ -301,15 +301,15 @@ def query():
         # Keep only pods created after current pod
         si = ip_list.index(ip)
         ip_list = ip_list[:si]
-        return [{"role": "inter-router", "host": host} for host in ip_list]
+        return [{"role": "inter-router", "host": "config-sync-ignore-%s" % host} for host in ip_list]
```

Comment:
I think we want to add a *name* with this pattern, but leave the host as it is (otherwise it won't resolve correctly).

Reply:
That makes a lot of sense, I will fix that, thank you!